lib/active_model/serializer.rb

```diff
@@ -2,10 +2,12 @@
 
 module ActiveModel
   class Serializer
+    Error = Class.new(StandardError)
     extend ActiveSupport::Autoload
     autoload :Configuration
     autoload :ArraySerializer
     autoload :Adapter
+    autoload :Builder
```

Comment:
autoload is threadsafe for some time now. See https://github.com/headius/thread_safe/issues/15#issuecomment-28936120 and even if it weren't `extend ActiveSupport::Autoload` would make it so.